src/Catalyst.Node.POA.CE.Tests/IntegrationTests/PoaTestNode.cs

```diff
@@ -72,6 +74,8 @@ public class PoaTestNode : ICatalystNode, IDisposable
         private readonly IRpcServerSettings _rpcSettings;
         private readonly ILifetimeScope _scope;
         private readonly ContainerProvider _containerProvider;
+        private readonly IDeltaByNumberRepository _deltaByNumber;
+        private readonly ITransactionReceiptRepository _receipts;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_receipts' or refactor the code to use its value.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4585091)